api/src/test/java/com/epam/pipeline/manager/docker/DockerContainerOperationManagerTest.java

```diff
@@ -109,21 +113,28 @@
     @Mock
     private PipelineConfigurationManager pipelineConfigurationManager;
 
+    @Mock
+    private MessageHelper messageHelper;
```

Comment:
Could you please fix `pmdTest` task? (add `@SuppressWarnings("PMD.UnusedPrivateField")` to field)